visualize-admin/visualization-tool
# Scientific notation for decimal

Fix : https://github.com/visualize-admin/visualization-tool/issues/690

For dimensions with decimal datatype, we use a scientific notation formatter, when the value is > 999 999 999 or value < 0.0001

## How to test

* See the greenhouse gas emissions (hierarchie) dataset

`/en/browse/dataset/https://environment.ld.admin.ch/foen/ubd000502test/3`

* The column "Greenhouse gas emissions in CO2eq(Mt) should be in scientific notation
* When looking at a chart, the tooltips should show number formatter in scientific notation
